cpu/atmega_common/Makefile.features

```diff
@@ -3,6 +3,10 @@ include $(RIOTCPU)/avr8_common/Makefile.features
 # common feature are defined in avr8_common/Makefile.features
 # Only add Additional features
 
+ifneq (,$(filter periph_rtt,$(FEATURES_PROVIDED)))
+  FEATURES_PROVIDED += periph_rtt_set_counter
+endif
```

Comment:
We can set this pseudofeature unconditionally, see #15763

Reply:
OK.